src/main/java/org/jabref/logic/exporter/GlobalSaveManager.java

```diff
@@ -0,0 +1,89 @@
+package org.jabref.logic.exporter;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.UnsupportedCharsetException;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.Future;
+import java.util.function.Consumer;
+
+import org.jabref.gui.Globals;
+import org.jabref.gui.StateManager;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.util.DelayTaskThrottler;
+import org.jabref.model.FieldChange;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.preferences.PreferencesService;
+
+public class GlobalSaveManager {
+
+    private static Set<GlobalSaveManager> runningInstances = new HashSet<>();
+
+    private final DelayTaskThrottler<Set<Character>> throttler = new DelayTaskThrottler<>(1500);
+    private final BibDatabaseContext bibDatabaseContext;
+
+    private final StateManager stateManager;
+
+    private final PreferencesService preferencesService;
+
+    private GlobalSaveManager(StateManager stateManager, PreferencesService preferencesService, BibEntryTypesManager entryTypesManager) {
+        this.bibDatabaseContext=  stateManager.getActiveDatabase().get();
+        this.stateManager = stateManager;
+        this.preferencesService = preferencesService;
+
+    }
+
+    public static void shutdown(BibDatabaseContext context) {
+        runningInstances.stream().filter(instance -> instance.bibDatabaseContext == context).forEach(GlobalSaveManager::shutdown);
+        runningInstances.removeIf(instance -> instance.bibDatabaseContext == context);
+    }
+
+    public static GlobalSaveManager create(StateManager stateManager, PreferencesService preferencesService, BibEntryTypesManager entryTypesManager) {
```

Comment:
the other methods (in backupmanager eg) are called "start", this should be aligned